functions.py

```diff
@@ -1,36 +1,35 @@
 import time
+
+def ReadLines(nameoffile):
+    with open(nameoffile,'r') as f:
+        arrayline = f.readlines()
+        return arrayline
+
 #how much is not taxed
 def SellWithoutTax():
-    f = open('new_register.txt','r')
-    try:
-        arrayline = f.readlines()
-        sum = 0.0
-        i = 0
+    arrayline = ReadLines('new_register.txt')
+    sum = 0.0
+    i = 0
+    arr = arrayline[i].split(',', 2)

```

Comment:
arr - variable name is not explicit, pls try to come up with a more explicit filename